compiler/tests/generated/service_grpc.h

```diff
@@ -178,12 +178,42 @@ class Foo final
             BOOST_ASSERT(cq0);
             BOOST_ASSERT(tp);
 
-            /* TODO: init for event foo11 */
-            /* TODO: init for event foo12 */
-            /* TODO: init for event foo12_impl */
-            /* TODO: init for event foo13 */
-            /* TODO: init for event foo14 */
-            /* TODO: init for event foo15 */
+            _rd_foo11.emplace(
+                this,
+                0,
+                cq0,
+                tp,
+                std::bind(&ServiceCore::foo11, this, std::placeholders::_1));
+            _rd_foo12.emplace(
+                this,
+                1,
+                cq0,
+                tp,
+                std::bind(&ServiceCore::foo12, this, std::placeholders::_1));
```

Comment:
`std::function` was the easiest way _for me_ to get everything to work. Refactoring everything that today uses a `std::function` to take callable object hasn't happened yet. Feel free to delay as many of those refactorings as you need to to a separate commit and PR.